### PR TITLE
Update for catalog

### PR DESCRIPTION
**Pull request by jamesgking**
Minor edits to Funding information to make consistent with text used in other softwares released by Blue Brain Project

### CODE REVIEW

**Review by pramodk — APPROVED** [review]
LGTM

**Comment by iomaganaris** [comment]
please retest

File: README.md [hunk 1]
```diff
@@ -373,6 +373,6 @@ You can see current [contributors here](https://github.com/BlueBrain/CoreNeuron/
...
+CoreNEURON is developed in a joint collaboration between the Blue Brain Project and Yale University. This work was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology, NIH grant number R01NS11613 (Yale University), the European Union Seventh Framework Program (FP7/20072013) under grant agreement n◦ 604102 (HBP) and the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreement n◦ 720270 (Human Brain Project SGA1), n◦ 785907 (Human Brain Project SGA2) and n◦ 945539 (Human Brain Project SGA3).
```
Comment (ohm314) [inline README.md:376]:
```suggestion
CoreNEURON is developed in a joint collaboration between the Blue Brain Project and Yale University. This work is supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology, NIH grant number R01NS11613 (Yale University), the European Union Seventh Framework Program (FP7/20072013) under grant agreement n◦ 604102 (HBP) and the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreement n◦ 720270 (Human Brain Project SGA1), n◦ 785907 (Human Brain Project SGA2) and n◦ 945539 (Human Brain Project SGA3).
```

**Review by ohm314 — CHANGES_REQUESTED** [review]
I think just this minor fix is needed

**Review by ohm314 — APPROVED** [review]
lgtm

**Pull request merged**